index.rst

```diff
@@ -26,11 +26,14 @@ other friends.  :doc:`You can join us </about/join>`.
 
 News
 ====
-.. toctree::
-   :maxdepth: 2
+.. include:: /news/index.rst
+   :start-line: 3
+   :end-line: 7
 
-   news/index
+.. toctree::
+   :maxdepth: 1
 
+   news/index.rst
```

Comment:
Not important, but `.rst` isn't generally needed when linking files internally (it always finds them)